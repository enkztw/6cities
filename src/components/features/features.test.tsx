import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Features from './features';


const features: string[] = [`Heating`, `Kitchen`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`];

it(`<Features /> should be rendered`, () => {
  const tree = renderer.create(<Features goods={features} />);

  expect(tree).toMatchSnapshot();
});
