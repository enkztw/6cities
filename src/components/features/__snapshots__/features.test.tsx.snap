// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Features /> should be rendered 1`] = `
<div
  className="property__inside"
>
  <h2
    className="property__inside-title"
  >
    What's inside
  </h2>
  <ul
    className="property__inside-list"
  >
    <li
      className="property__inside-item"
    >
      Heating
    </li>
    <li
      className="property__inside-item"
    >
      Kitchen
    </li>
    <li
      className="property__inside-item"
    >
      Cable TV
    </li>
    <li
      className="property__inside-item"
    >
      Washing machine
    </li>
    <li
      className="property__inside-item"
    >
      Coffee machine
    </li>
    <li
      className="property__inside-item"
    >
      Dishwasher
    </li>
  </ul>
</div>
`;
